app/jobs/risc_delivery_job.rb

```diff
@@ -6,6 +6,7 @@ class RiscDeliveryJob < ApplicationJob
   NETWORK_ERRORS = [
     Faraday::TimeoutError,
     Faraday::ConnectionFailed,
+    Faraday::ServerError,
```

Comment:
is this to address the `UspsAuthTokenRefreshJob` errors we've been seeing? If so, that job does not reference this constant so it won't have any effect

Reply:
ok i think i;m missing something then. The def I found for `warning_error_classes` referenced `NETWORK_ERRORS` but i see now i didn't carefully check which job was using that. Where should i be looking instead?

Reply:
directly in that job:
https://github.com/18F/identity-idp/blob/c7f7d2ebbe5d45a67a23272e5c6bc96472e42853/app/jobs/usps_auth_token_refresh_job.rb#L10

Reply:
alternatively, the `UspsAuthTokenRefreshJob` could override its `warning_error_classes` and implement its own list, similar to the RiscDeliveryJob:

https://github.com/18F/identity-idp/blob/c7f7d2ebbe5d45a67a23272e5c6bc96472e42853/app/jobs/risc_delivery_job.rb#L22-L24